.github/workflows/ci.yml

```diff
@@ -109,7 +109,7 @@ jobs:
   compatibility-test:
     strategy:
       matrix:
-        go-version: [1.15, 1.14]
+        go-version: [1.16, 1.15, 1.14]
```

Comment:
Should we add a comment here detailing the rationale for the versions in this list?  I think we want to support 1.14 as a minimum (unless/until we decide a newer version adds a feature worth increasing that minimum supported version) and then test against the two latest versions.  I thought we had specified the two latest versions as official support, but I don't really see that documented.  I'd like to avoid having this list grow endlessly as new Go versions are released.  When 1.17 is released we should be able to drop 1.15.

Reply:
@Aneurysm9 Go support policy seems to be current and previous version only
https://golang.org/doc/devel/release.html#policy